mne/epochs.py

```diff
@@ -2180,6 +2191,22 @@ def read_epochs(fname, proj=True, add_eeg_ref=True, verbose=None):
 
     epochs = _concatenate_epochs(epochs, read_file=True)
     epochs._bad_dropped = True
+    data = epochs._data
+    # baseline correct
+    if baseline is None and epochs.baseline is not None:
+        raise ValueError('A baseline correction was already applied to the '
+                         'epochs, `baseline=None` is invalid.')
+    elif isinstance(baseline, tuple):
+        # Baseline correct
+        picks = pick_types(epochs.info, meg=True, eeg=True, stim=False,
+                           ref_meg=True, eog=True, ecg=True,
+                           emg=True, exclude=[])
+        data[:, picks, :] = rescale(data[:, picks, :], epochs.times, baseline,
+                                    'mean', copy=False, verbose=verbose)
+        epochs.baseline = baseline
+    elif isinstance(baseline, str) and baseline != 'original':
+        raise ValueError('`baseline=%s` is an invalid argument.' % baseline)
+
```

Comment:
For simplicity you can make it only operate if data are preloaded for now. A subsequent PR could be made to make it operate on non-preloaded data in theory, but we can assume YAGNI until we see otherwise.